math.js

```diff
@@ -20,4 +20,8 @@ export function sqrt(a) {
 
 export function log(a) {
     return Math.log(a);
-}
\ No newline at end of file
+}
+
+export function sin(a) {
```

Comment:
`a` is not a very well named parameter.  Did you consider a longer more expressive name?